rebar.config

```diff
@@ -61,14 +62,20 @@
 
 %% == Dependencies ==
 
-{deps, [ {xref_runner, "1.1.1"}
-       , {elvis_core, "0.4.1"}
+{deps, [ {xref_runner, "1.2.0"}
+       , {elvis_core, "1.1.2"}
        ]}.
 
+{project_plugins, [
+    rebar3_hank,
+    rebar3_hex,
+    rebar3_lint
+]}.
+
 %% == Dialyzer ==
 
 {dialyzer, [ {warnings, [ no_return
-                        , unmatched_returns
+                        %, unmatched_returns
```

Comment:
To be honest… we need to eventually get rid of `xref_runner` altogether… `rebar3` has improved so much since that was needed, that there is no point to keep it alive anymore.